TWLight/resources/templates/resources/partner_detail.html

```diff
@@ -219,6 +219,40 @@ <h1>{{ object }}</h1>
         </div>
       </div>      
     </li>
+    {% with object.excerpt_limit as excerpt_limit and object.excerpt_limit_percentage as excerpt_limit_percentage %}
+    {% if excerpt_limit or excerpt_limit_percentage %}
+      <li class="timeline-inverted">
+        <div class="timeline-badge"><i class="fa fa-files-o" aria-hidden="true"></i>
+        </div>
+        <div class="timeline-panel">
+          <div class="timeline-heading">
+            {% comment %} Translators: If a partner has specified the excerpt limit, this text shows as the title for that information field. {% endcomment %}
+            <h4 class="timeline-title">{% trans "Excerpt limit" %}</h4>
+          </div>        
+          <div class="timeline-body">
+            <ul>
+              {% if excerpt_limit %}
```

Comment:
Could you add a 3rd option for "...{{ excerpt_limit }} words or {{ excerpt_limit_percentage }} of an article be excerpted...

This will save space on the partner pages when both have values.